### ISSUE TITLE
BruTile iOS build version issue

### ISSUE THREAD

**Issue by FilipNemec** [Mapsui/Mapsui#2217]
**Mapsui Version**
4.0.0

**Mapsui Platform**
Xamarin

**Device**
iOS

**Describe the bug**
In my project I have include BruTile.MbTiles v5.0.5. The same version is also included in Mapsui packages. When building iOS app with linker settings set to "None" the build is successful. Building with "Framework SDK's only" and adding explicitly to preserve BruTile assembly I get the fallowing error: 

> MTOUCH : error MT2101: Can't resolve the reference 'System.Void BruTile.Web.HttpTileSource::.ctor(BruTile.ITileSchema,System.String,System.Collections.Generic.IEnumerable`1<System.String>,System.String,System.String,BruTile.Cache.IPersistentCache`1<System.Byte[]>,System.Func`2<System.Uri,System.Threading.Tasks.Task`1<System.Byte[]>>,BruTile.Attribution,System.String)', referenced from the method 'BruTile.Web.HttpTileSource Mapsui.Tiling.OpenStreetMap::CreateTileSource(System.String)' in 'BruTile, Version=5.0.4.0, Culture=neutral, PublicKeyToken=null'. [/Users/admin/BuildAgent/13/work/fbe61631739df768/Name1.Name2/Name1.Name2.iOS/Name1.Name2.iOS.csproj]

In my project I don't have a reference to BruTile v5.0.4. It seams that the issue is coming from Mapsui packages. I saw a similar issue in 2019 https://github.com/Mapsui/Mapsui/issues/703

**Comment by pauldendulk** [comment]
ai, that is my mistake. I introduced a breaking change in BruTile 5.0.5. I added an optional parameter to that method, thinking since it is optional no one will notice, but if you do not compile against that code you get the error  above. Mapsui expects the exact same signature for that method. I should have introduced a separate method for that purpose instead. My plan is currently not to fix it since there are solutions.

- Use Mapsui 4.1.0 (released an hour ago).
- Use BruTile 5.0.4 in your own project.

Tell me if this fixes your problem.
